src/vtarget-out/gen_util.cc

```diff
@@ -117,8 +117,9 @@ VlgSglTgtGen::ModifyCondExprAndRecordVlgName(const VarExtractor::token& t) {
   const auto& sname = t.second;
 
   if (token_tp == VarExtractor::token_type::UNKN_S) {
-    ILA_WARN << "In refinement relations: unknown reference to name:" << sname
-             << " keep unchanged.";
+    ILA_WARN_IF( ! ( sname == "__IEND__" || sname == "__START__") ) 
```

Comment:
Will adding `__CYCLE_CNT__` here cause any problem?

Reply:
Let's add it.